packages/tinacms/src/toolkit/fields/components/reference/utils/fetch-optioins-query-builder.ts

```diff
@@ -0,0 +1,22 @@
+// This type is used in collectionFilter for reference field, it represent the datatype of the field in the referenced collection that will be used to do the filter
+// This type is extendable if we need to support more type for reference selection filter (e.g. boolean)
```

Comment:
**suggestion**: Improved English
```suggestion
// It is extendable in case we need to support more types for the reference selection filter (e.g. boolean)
```